src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -13,7 +13,106 @@
 #               2. compute f(u, grad(u))
 #               3. compute div(f(u, grad(u))) (i.e., the "regular" rhs! call)
 # boundary conditions will be applied to both grad(u) and div(f(u, grad(u))).
-function rhs_parabolic!(du, u, t, mesh::Union{TreeMesh{2}, P4estMesh{2}},
+function rhs_parabolic!(du, u, t, mesh::TreeMesh{2},
```

Comment:
I added this dispatched version since otherwise some tests fail, see 

https://github.com/trixi-framework/Trixi.jl/actions/runs/5579096780/jobs/10194198147